guides/lambda-configuration.md

```diff
@@ -3,15 +3,20 @@
 This covers how to configure and enable AppOptics APM for nodejs lambda functions. For
 general configuration options see `configuration.md` in the same directory.
 
-## Prerequisite
+## Prerequisites
+
+If you haven't already, set up the forwarder. This only needs to be done once per region
+for the account.
 
 Add the `appoptics-node` layer to your function
-- screen shots?
-- select `arn:aws:lambda:${your-function's-region}:858939916050:layer:appoptics-node`:41
-- forwarder setup already done (where to document this - each agent?)
+- find the current version for your region(s) at https://files.appoptics.com/lambda-layers/node/
+- copy the arn (e.g., `arn:aws:lambda:us-east-1:085151004374:layer:appoptics-node:7`)
```

Comment:
In the KB docs draft for the [Node layer](https://docs.google.com/document/d/1s63i3_NZn-H6r7Wyb4ObercjWxL3292dbkCeWJO3dR4/edit#heading=h.lddh8ahnezo2) we're calling it `appoptics-node`, with the index file located under https://files.appoptics.com/lambda-layers/appoptics-node 

Also clicking the file for us-east-1 (https://files.appoptics.com/lambda-layers/node/us-east-1) downloaded rather than displayed it, is there some way to get it to display like https://files.appoptics.com/lambda-layers/appoptics-apm-layer instead?

And noticed that the downloaded file for us-east-1 has the layer name as `appoptics-apm-layer` instead of `appoptics-node`
```arn:aws:lambda:us-east-1:085151004374:layer:appoptics-apm-layer:5 - apm v9.0.0-lambda-27, bindings v10.0.0-lambda-8, auto v1.0.1-lambda-3```

Reply:
> with the index file located under https://files.appoptics.com/lambda-layers/node. 

this is just a test. the name can be anything desired. nice catch in the docs.

> is there some way to get it to display like https://files.appoptics.com/lambda-layers/appoptics-apm-layer instead?

i was originally going to ask you about this. it turns out that making files appear requires running the `update-prod-index` jenkins job, making sure they have correct permissions is another step (need to add ACL public-read), and whether the file is downloadable or not is unknown to me at this time; i don't have console access to the production s3 bucket so it is a black box. the reason that the files were published under node is that i didn't know whether we had permissions to delete keys or not and i didn't want to create any that couldn't be deleted; it turns out i can delete keys so it wasn't an issue.

so hopefully you know why it wants to download the file. looking at each using `aws s3api` they are both text/plain at this time and yet us-west-2 displays while us-east-1 offers a download.

> And noticed that the downloaded file for us-east-1 has the layer name as appoptics-apm-layer

this is because we don't want to publish any `appoptics-node` layers in production until i publish a real one. publishing `appoptics-node` to production will use up version numbers in the production layer, so these files were run against a different layer name to test. i can publish appoptics-node layers in the apm-instrumentation account and write those to the production files area if you'd like to see that.

Reply:
Looks like the display is working now 👍